website/content/en/preview/upgrade-guide.md

```diff
@@ -106,6 +106,10 @@ Snapshot releases are tagged with the git commit hash prefixed by the Karpenter
 * Karpenter has changed the default metrics service port from 8080 to 8000. In `v0.28.0`, the Karpenter pod port was changed to 8000, but referenced the service by name, allowing users to scrape the service at port 8080 for metrics. `v0.29.0` aligns the two ports so that default service and default pod metrics ports are the same.
```

Comment:
You also need to propagate this change over into the `docs/` directory so that it shows on the latest release version docs as well